votes/0544-sig-txn-reviewer-to-tidb.md

```diff
@@ -0,0 +1,27 @@
+# A Vote for the membership of prev SIG-Transaction's reviewers
+
+## Proposal
+
+Add @ekexium, @longfangsong and @you06 as TiDB reviewers, all of whom are reviewers of the previous SIG-Transaction.
+
+@ekexium has contributions to the async commit project and is responsible for the the data consistency issues.
+* [Authored pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3Aekexium)
+
+@longfangsong has great contributions made to TiDB's lock views.
+* [Authored pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3Alongfangsong)
+
+@you06 has had a lot of bug fixes in various modules in TiDB.
+* [Authored pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+author%3Ayou06).
+
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
```

Comment:
```suggestion
The vote will be open for at least 3 days unless there is an objection or not enough votes.
```

according to https://github.com/pingcap/community/blob/master/teams/README.md#actions